### ISSUE TITLE
Main interface/Panels - Performances

### ISSUE THREAD

**Issue by Grum999** [Grum999/BuliCommander#23]
From KA forum: https://krita-artists.org/t/bulicommander/11612/34?u=grum999

Starting to look in detail execution time:
- Open window plugin from directory X: ~2.28s
- Change directory to X: ~0.98s

### Startup 

``` text 
DEBUG: >> Start: BCMainWindow.01.00.init
DEBUG: >> Start: BCMainWindow.01.01.init-ui
DEBUG: >> Start: BCMainViewTab[0x7f8f5729d670].01.99.init
DEBUG: >> Start: BCMainViewTab[0x7f8f5729d670].01.01.init-var
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729d670].01.01.init-var -- (0.00280499  @+0.01607823)
DEBUG: >> Start: BCMainViewTab[0x7f8f5729d670].01.02.init-ui
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729d670].01.02.init-ui -- (0.12602472  @+0.14213943)
DEBUG: >> Start: BCMainViewTab[0x7f8f5729d670].01.03.init-misc
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729d670].01.03.init-misc -- (0.01798463  @+0.16016626)
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729d670].01.99.init -- (0.14694500  @+0.16020155)
DEBUG: >> Start: BCMainViewTab[0x7f8f5729df70].01.99.init
DEBUG: >> Start: BCMainViewTab[0x7f8f5729df70].01.01.init-var
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729df70].01.01.init-var -- (0.00040483  @+0.16159034)
DEBUG: >> Start: BCMainViewTab[0x7f8f5729df70].01.02.init-ui
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729df70].01.02.init-ui -- (0.07224846  @+0.23386097)
DEBUG: >> Start: BCMainViewTab[0x7f8f5729df70].01.03.init-misc
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729df70].01.03.init-misc -- (0.01197386  @+0.24586987)
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729df70].01.99.init -- (0.08471298  @+0.24588680)
DEBUG: << Stop:  BCMainWindow.01.01.init-ui -- (0.68344665  @+0.68351483)
DEBUG: >> Start: BCMainWindow.01.02.init-var
DEBUG: << Stop:  BCMainWindow.01.02.init-var -- (0.02447295  @+0.70802069)
DEBUG: << Stop:  BCMainWindow.01.00.init -- (0.70799017  @+0.70803857)
DEBUG: >> Start: BCUIController.01.99.__initSettings
DEBUG: >> Start: BCUIController.01.01.__initSettings
DEBUG: >> Start: BCMainWindow.02.00.initMainView
DEBUG: << Stop:  BCMainWindow.02.00.initMainView -- (0.01224232  @+0.74112058)
DEBUG: << Stop:  BCUIController.01.01.__initSettings -- (0.01230788  @+0.74114633)
DEBUG: >> Start: BCUIController.01.02.__initSettings
DEBUG: << Stop:  BCUIController.01.02.__initSettings -- (0.00000501  @+0.74116278)
DEBUG: >> Start: BCUIController.01.03.__initSettings
DEBUG: >> Start: BCMainViewTab[0x7f8f5729d670].filesPath_Changed.01.99
DEBUG: >> Start: BCMainViewTab[0x7f8f5729d670].filesPath_Changed.01.01
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729d670].filesPath_Changed.01.01 -- (0.00000429  @+1.10067368)
DEBUG: >> Start: BCMainViewTab[0x7f8f5729d670].filesPath_Changed.01.02
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729d670].filesPath_Changed.01.02 -- (0.00002956  @+1.10071540)
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729d670].filesPath_Changed.01.99 -- (0.00053740  @+1.10072351)
DEBUG: >> Start: BCMainViewTab[0x7f8f5729df70].filesPath_Changed.01.99
DEBUG: >> Start: BCMainViewTab[0x7f8f5729df70].filesPath_Changed.01.01
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729df70].filesPath_Changed.01.01 -- (0.00000691  @+1.21730828)
DEBUG: >> Start: BCMainViewTab[0x7f8f5729df70].filesPath_Changed.01.02
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729df70].filesPath_Changed.01.02 -- (0.00002718  @+1.21735001)
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729df70].filesPath_Changed.01.99 -- (0.00048923  @+1.21735787)
DEBUG: << Stop:  BCUIController.01.03.__initSettings -- (0.53420448  @+1.27538252)
DEBUG: >> Start: BCUIController.01.04.__initSettings
DEBUG: << Stop:  BCUIController.01.04.__initSettings -- (0.00384974  @+1.27926517)
DEBUG: >> Start: BCUIController.01.05.__initSettings
DEBUG: << Stop:  BCUIController.01.05.__initSettings -- (0.00016642  @+1.27945542)
DEBUG: >> Start: BCUIController.01.06.__initSettings
DEBUG: >> Start: BCMainViewTab[0x7f8f5729d670].__filesRefresh.01
DEBUG: ===========================================
DEBUG: BCFileList.execute
DEBUG: ...........................................
DEBUG: Search in paths: [<BCFileListPath('/home/grum/Data/Temp/testmd5', False)>]
DEBUG: Found 500 of 500 files in 0.01070857048034668s
DEBUG: - Files: 500
DEBUG: - Directories: 0
DEBUG: >> Start: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init
DEBUG: << Stop:  WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init -- (0.00093722  @+1.30013943)
DEBUG: >> Start: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute
DEBUG: << Stop:  WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute -- (0.12679362  @+1.42696834)
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Processed 500 items with 2 workers
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute: 0.12679362
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init: 0.00093722
DEBUG: ===========================================
DEBUG: Filter 500 files to 500 files in 0.1286461353302002s
DEBUG: >> Start: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init
DEBUG: << Stop:  WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init -- (0.00012541  @+1.42807961)
DEBUG: >> Start: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute
DEBUG: << Stop:  WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute -- (0.01892900  @+1.44703436)
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Processed 500 items with 2 workers
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute: 0.01892900
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init: 0.00012541
DEBUG: ===========================================
DEBUG: >> Start: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init
DEBUG: << Stop:  WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init -- (0.00005937  @+1.44839334)
DEBUG: >> Start: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute
DEBUG: << Stop:  WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute -- (0.01685643  @+1.46526504)
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Processed 500 items with 2 workers
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute: 0.01685643
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init: 0.00005937
DEBUG: ===========================================
DEBUG: BCFileList.execute.01-search: 0.01070857
DEBUG: BCFileList.execute.02-filter: 0.12864614
DEBUG: BCFileList.execute.03-result: 0.02039981
DEBUG: BCFileList.execute.04-buildStats: 0.01811695
DEBUG: BCFileList.execute.05-sort: 0.00000477
DEBUG: BCFileList.execute.99-global: 0.17827249
DEBUG: ===========================================
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729d670].__filesRefresh.01 -- (0.18720889  @+1.46670890)
DEBUG: >> Start: BCFileModel[0x7f8f5729d940].__dataUpdateSort
DEBUG: << Stop:  BCFileModel[0x7f8f5729d940].__dataUpdateSort -- (0.00320649  @+1.49513841)
DEBUG: >> Start: BCFileModel[0x7f8f5729db80].__dataUpdateSort
DEBUG: << Stop:  BCFileModel[0x7f8f5729db80].__dataUpdateSort -- (0.00322151  @+1.49837637)
DEBUG: << Stop:  BCUIController.01.06.__initSettings -- (0.87273455  @+2.15220380)
DEBUG: >> Start: BCUIController.01.07.__initSettings
DEBUG: << Stop:  BCUIController.01.07.__initSettings -- (0.01808476  @+2.17030978)
DEBUG: << Stop:  BCUIController.01.99.__initSettings -- (1.44156599  @+2.17032361)
``` 

### Change directory 
```text
DEBUG: >> Start: BCMainViewTab[0x7f8f5729d670].filesPath_Changed.01.99
DEBUG: >> Start: BCMainViewTab[0x7f8f5729d670].filesPath_Changed.01.01
DEBUG: >> Start: BCMainViewTab[0x7f8f5729d670].__filesRefresh.01
DEBUG: ===========================================
DEBUG: BCFileList.execute
DEBUG: ...........................................
DEBUG: Search in paths: [<BCFileListPath('/home/grum/Data/Temp', False)>]
DEBUG: Found 2 of 2 files in 0.0032536983489990234s
DEBUG: - Files: 0
DEBUG: - Directories: 2
DEBUG: >> Start: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init
DEBUG: << Stop:  WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init -- (0.00006485  @+0.03051114)
DEBUG: >> Start: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute
DEBUG: << Stop:  WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute -- (0.00513172  @+0.03565860)
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Processed 2 items with 2 workers
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute: 0.00513172
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init: 0.00006485
DEBUG: ===========================================
DEBUG: Filter 2 files to 0 files in 0.007854938507080078s
DEBUG: >> Start: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init
DEBUG: << Stop:  WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init -- (0.00009227  @+0.03849816)
DEBUG: >> Start: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute
DEBUG: << Stop:  WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute -- (0.00090194  @+0.03941655)
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Processed 2 items with 2 workers
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute: 0.00090194
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init: 0.00009227
DEBUG: ===========================================
DEBUG: >> Start: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init
DEBUG: << Stop:  WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init -- (0.00005913  @+0.04062867)
DEBUG: >> Start: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute
DEBUG: << Stop:  WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute -- (0.00026751  @+0.04091001)
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Processed 2 items with 2 workers
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute: 0.00026751
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init: 0.00005913
DEBUG: ===========================================
DEBUG: BCFileList.execute.01-search: 0.00325370
DEBUG: BCFileList.execute.02-filter: 0.00785494
DEBUG: BCFileList.execute.03-result: 0.00217199
DEBUG: BCFileList.execute.04-buildStats: 0.00148511
DEBUG: BCFileList.execute.05-sort: 0.00000548
DEBUG: BCFileList.execute.99-global: 0.01508451
DEBUG: ===========================================
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729d670].__filesRefresh.01 -- (0.01538610  @+0.04220343)
DEBUG: >> Start: BCFileModel[0x7f8f5729d940].__dataUpdateSort
DEBUG: << Stop:  BCFileModel[0x7f8f5729d940].__dataUpdateSort -- (0.00720882  @+0.04965401)
DEBUG: >> Start: BCFileModel[0x7f8f5729db80].__dataUpdateSort
DEBUG: << Stop:  BCFileModel[0x7f8f5729db80].__dataUpdateSort -- (0.00121999  @+0.05089617)
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729d670].filesPath_Changed.01.01 -- (0.08555675  @+0.11235952)
DEBUG: >> Start: BCMainViewTab[0x7f8f5729d670].filesPath_Changed.01.02
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729d670].filesPath_Changed.01.02 -- (0.00003886  @+0.11242580)
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729d670].filesPath_Changed.01.99 -- (0.08572698  @+0.11243534)
DEBUG: >> Start: BCMainViewTab[0x7f8f5729d670].filesPath_Changed.01.99
DEBUG: >> Start: BCMainViewTab[0x7f8f5729d670].filesPath_Changed.01.01
DEBUG: >> Start: BCMainViewTab[0x7f8f5729d670].__filesRefresh.01
DEBUG: ===========================================
DEBUG: BCFileList.execute
DEBUG: ...........................................
DEBUG: Search in paths: [<BCFileListPath('/home/grum/Data/Temp/testmd5', False)>]
DEBUG: Found 500 of 500 files in 0.005563497543334961s
DEBUG: - Files: 500
DEBUG: - Directories: 0
DEBUG: >> Start: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init
DEBUG: << Stop:  WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init -- (0.00057912  @+0.03675056)
DEBUG: >> Start: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute
DEBUG: << Stop:  WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute -- (0.14278078  @+0.17955852)
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Processed 500 items with 2 workers
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute: 0.14278078
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init: 0.00057912
DEBUG: ===========================================
DEBUG: Filter 500 files to 500 files in 0.144622802734375s
DEBUG: >> Start: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init
DEBUG: << Stop:  WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init -- (0.00009561  @+0.18100595)
DEBUG: >> Start: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute
DEBUG: << Stop:  WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute -- (0.02333617  @+0.20436239)
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Processed 500 items with 2 workers
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute: 0.02333617
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init: 0.00009561
DEBUG: ===========================================
DEBUG: >> Start: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init
DEBUG: << Stop:  WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init -- (0.00005960  @+0.20569110)
DEBUG: >> Start: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute
DEBUG: << Stop:  WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute -- (0.01848555  @+0.22419667)
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Processed 500 items with 2 workers
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Execute: 0.01848555
DEBUG: WorkerPool.{e3b30cda-f0a3-489b-b9e0-0f57ca672c1f}.Init: 0.00005960
DEBUG: ===========================================
DEBUG: BCFileList.execute.01-search: 0.00556350
DEBUG: BCFileList.execute.02-filter: 0.14462280
DEBUG: BCFileList.execute.03-result: 0.02474046
DEBUG: BCFileList.execute.04-buildStats: 0.01975489
DEBUG: BCFileList.execute.05-sort: 0.00000739
DEBUG: BCFileList.execute.99-global: 0.19499040
DEBUG: ===========================================
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729d670].__filesRefresh.01 -- (0.19524097  @+0.22555137)
DEBUG: >> Start: BCFileModel[0x7f8f5729d940].__dataUpdateSort
DEBUG: << Stop:  BCFileModel[0x7f8f5729d940].__dataUpdateSort -- (0.00671434  @+0.25686765)
DEBUG: >> Start: BCFileModel[0x7f8f5729db80].__dataUpdateSort
DEBUG: << Stop:  BCFileModel[0x7f8f5729db80].__dataUpdateSort -- (0.00304866  @+0.25993586)
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729d670].filesPath_Changed.01.01 -- (0.91623616  @+0.94653130)
DEBUG: >> Start: BCMainViewTab[0x7f8f5729d670].filesPath_Changed.01.02
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729d670].filesPath_Changed.01.02 -- (0.00010061  @+0.94665956)
DEBUG: << Stop:  BCMainViewTab[0x7f8f5729d670].filesPath_Changed.01.99 -- (0.91646218  @+0.94666839)
``` 

## Some analysis 

- UI creation took 0.68s ***(it's long!)***
  `DEBUG: << Stop:  BCMainWindow.01.01.init-ui -- (0.68344665  @+0.68351483)`
- Panels initialisation took 0.53s ***(it's long!)***
  `DEBUG: << Stop:  BCUIController.01.03.__initSettings -- (0.53420448  @+1.27538252)`
- File search+analysis+filter+sort took 0.18s ***(it's fast)***
  `DEBUG: << Stop:  BCMainViewTab[0x7f8f5729d670].__filesRefresh.01 -- (0.18720889  @+1.46670890)`
- Something happen during initialisation step 06 (include *File search+analysis+filter+sort*)
  `DEBUG: << Stop:  BCUIController.01.06.__initSettings -- (0.87273455  @+2.15220380)`
  There 0.68s (2.15220380 - 1.46670890) spent between end of panel udpdate & plugin init ***(it's long!)***

These point need to be analyzed deeper to understand from where came this duration and try to reduce them

**Comment by Grum999** [comment]
Something happen during initialisation step 06 (include File search+analysis+filter+sort)
`DEBUG: << Stop: BCUIController.01.06.__initSettings -- (0.87273455 @+2.15220380)`
There 0.68s (2.15220380 - 1.46670890) spent between end of panel udpdate & plugin init (it's long!)

Looking deeper on this part:
```text
DEBUG: >> Start: BCMainViewTab[0x7f7b4a1fb310].__filesSort.04
DEBUG: >> Start: BCMainViewTab[0x7f7b4a1fb310].__filesUpdate.01
DEBUG: << Stop:  BCMainViewTab[0x7f7b4a1fb310].__filesUpdate.01 -- (0.34558105  @+1.77923918)
DEBUG: >> Start: BCMainViewTab[0x7f7b4a1fb310].__filesUpdate.02
DEBUG: << Stop:  BCMainViewTab[0x7f7b4a1fb310].__filesUpdate.02 -- (0.00030279  @+1.77957153)
DEBUG: >> Start: BCMainViewTab[0x7f7b4a1fb310].__filesUpdate.03
DEBUG: << Stop:  BCMainViewTab[0x7f7b4a1fb310].__filesUpdate.03 -- (0.00006604  @+1.77964926)
DEBUG: >> Start: BCMainViewTab[0x7f7b4a1fb310].__filesUpdate.04
DEBUG: << Stop:  BCMainViewTab[0x7f7b4a1fb310].__filesUpdate.04 -- (0.00004601  @+1.77970910)
DEBUG: >> Start: BCMainViewTab[0x7f7b4a1fb310].__filesUpdate.05
DEBUG: << Stop:  BCMainViewTab[0x7f7b4a1fb310].__filesUpdate.05 -- (0.31584215  @+2.09556603)
DEBUG: << Stop:  BCMainViewTab[0x7f7b4a1fb310].__filesSort.04 -- (0.66194677  @+2.09559321)
```

There's 2 actions for which time execution is high:
- `DEBUG: << Stop:  BCMainViewTab[0x7f7b4a1fb310].__filesUpdate.01 -- (0.34558105  @+1.77923918)` 
  Match with call to `treeViewFiles.setShowPath()`
- `DEBUG: << Stop:  BCMainViewTab[0x7f7b4a1fb310].__filesUpdate.05 -- (0.31584215  @+2.09556603)`
  Match with call to `treeViewFiles.resizeColumns()`

Need to take a look on it...

**Comment by Grum999** [comment]
There's 2 actions for which time execution is high:

- `DEBUG: << Stop: BCMainViewTab[0x7f7b4a1fb310].__filesUpdate.01 -- (0.34558105 @+1.77923918)`
    Match with call to `treeViewFiles.setShowPath()`
- `DEBUG: << Stop: BCMainViewTab[0x7f7b4a1fb310].__filesUpdate.05 -- (0.31584215 @+2.09556603)`
    Match with call to `treeViewFiles.resizeColumns()`

Problem fixed:
- `setShowPath()` implies call to `resizeColumns()` then `resizeColumns()` is called twice
  - Remove all to `resizeColumns()`, only keep `setShowPath()` (reduce execution time by 2)
- `resizeColumns()` calculate size of columns even if not visible; do calculation for visible columns only to reduce execution time (note: if all columns are visible, then this execution time is not reduced...)

**Comment by Grum999** [comment]
Made some small improvement on panels initialization

## Before change
DEBUG: << Stop: BCUIController.01.03.__initSettings -- (0.53420448 @+1.27538252)

## After change
DEBUG: << Stop:  BCUIController.01.03.__initSettings -- (0.31304741  @+1.05229306)

Not a big change, but sum all small change can make a difference at the end :) 

Here it will be hard to get more reduced initialization time..

**Comment by Grum999** [comment]
Made improvement on UI initialization

## Before change
`DEBUG: << Stop: BCMainWindow.01.01.init-ui -- (0.68344665 @+0.68351483)`

## After change
`DEBUG: << Stop: BCMainWindow.01.01.init-ui -- (0.24840045  @+0.24846768)`

**Comment by Grum999** [comment]
Taking in account all change 

## Before
`DEBUG: << Stop:  BCUIController.01.99.__initSettings -- (1.44156599  @+2.17032361)`

## After 
`DEBUG: << Stop:  BCUIController.01.99.__initSettings -- (0.57420158  @+0.85760641)`

## Synthesis
Initialization time reduced from 2.17s to 0.85s 🙂

**Comment by Grum999** [comment]
Will be closed with release v0.9.1b
